simon/_posts/2017-10-16-day-33.md

```diff
@@ -12,5 +12,8 @@ Things to write about:
 * How many books did you finish since your last journal entry? **Record the number** and write what they were.
 * Did you do any music or crafts?
 
-***
+I read creation and wrote about these puncuation marks (here are the puncation marks. ?!.). Then i corected my math and workedon my spelling . Then i made a sheduale.
 
+I finished 2 books since my last journal entry. One called **Winnie The Poo** the other called **The Last Olympian.**.
```

Comment:
*What about a list?*

I finished 2 books since my last journal entry:

1. **Winnie the Pooh** by A. A. Milne
2. **The Last Olympian** by Rick Riordan